src/cryptoadvance/specter/templates/includes/tx-table.html

```diff
@@ -776,6 +776,11 @@ <h1>{{ _("Export transactions to CSV") }}</h1>
         */
         attributeChangedCallback(attrName, oldValue, newValue) {
             if (attrName != "selected-coins") {
+                if (this.getAttribute('type') == "txlist") {
+                    this.amountHeader.innerText="Flow Amount"
```

Comment:
```suggestion
                    this.amountHeader.innerText="Amount \u0394"
```

Reply:
Both electrum and bitcoin core call this "Amount".